### ISSUE TITLE
Inconsistent capitalization of input params

### ISSUE THREAD

**Issue by gsgall** [shannon-lab/zapdos#214]
## Issue 

Some objects require inputs where the first letter is capital and others do not. This should be consistent or a convection for capitalization vs non-capitalization should be established for input parameters. 

Ex. `test/tests/DrifDiffusionAction/RF_Plasma_actions.i`
```
[DriftDiffusionAction]
  [Plasma]
    electrons = em
    charged_particle = Ar+
    Neutrals = Ar*
    mean_energy = mean_en
    potential = potential
    Is_potential_unique = true
    using_offset = false
    position_units = ${dom0Scale}
    Additional_Outputs = 'ElectronTemperature Current EField'
  []
[]
```

## Objects with issues 

1. DriftDiffusionAction 
2. PeriodicControllers 
3. DensityNormalization
4. TM0CylindricalErAux
5. TM0CylindricalEzAux
6. CircuitDirchletPotential
7. DriftDiffusionDoNothingBC
8. MatchedValueLogBC
9. SakiyamaEnergySecondaryElectronBC
10. ArbitraryTiedValueConstraint
11. DriftDiffusion
12. ElectronsFromIonization
13. TM0CylindricalEr
14. TM0CylindricalEz
15. SideCurrent

**Comment by csdechant** [comment]
This issue should be closed once PR #310 is merged.
@gsgall my questions regarding the mention objects still stands, but it is possible they were fixed in another PR that forgot to reference this issue.
